src/components/cart/CartTotalBox/CartTotalBox.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+import "./CartTotalBox.css";
+
+const CartTotalBox = () => {
+  return (
+    <div class="total-box">
+      <div class="cart-total-box">
+        <p class="p-large text-left text-bold-weight">Price Details</p>
+        <hr />
+        <div class="side-ways-cart">
+          <p class="p-large text-left">Price (2 items)</p>
+          <p class="p-large text-left">Rs. 1198</p>
+        </div>
+        <div class="side-ways-cart">
+          <p class="p-large text-left">Discount</p>
+          <p class="p-large text-left">- Rs. 120</p>
+        </div>
+        <div class="side-ways-cart">
+          <p class="p-large text-left">Delivery charges</p>
+          <p class="p-large text-left">Rs. 0</p>
+        </div>
+        <hr />
+        <div class="side-ways-cart">
+          <p class="p-large text-left text-bold-weight">TOTAL AMOUNT</p>
```

Comment:
i guess here **text align** can give to parent so that it will applies to its child's